polaris-react/src/components/InContextLearning/InContextLearning.tsx

```diff
@@ -60,6 +70,13 @@ export function InContextLearning({children, steps, onDismiss}: Props) {
     </div>
   );
 
+  const counterMarkup = hasMultipleSteps ? (
+    <p style={{textAlign: 'center'}}>
```

Comment:
I have two `textAlign` inline styles to align the counter and "Back" button correctly. There is maybe a Polaris way to do this as well? Or eventually we should have a PR to move all these inline styles to a stylesheet.